client/src/App.tsx

```diff
@@ -30,6 +31,7 @@ function App(): JSX.Element {
                   <Route exact path="/signup" component={Signup} />
                   <Route exact path="/listings" component={Listings} />
                   <ProtectedRoute exact path="/user/:path" component={ProfileSettings} />
+                  <ProtectedRoute exact path="/dashboard" component={Dashboard} />
```

Comment:
I added it as a route prior to the creation of FE Messages UI, for the Dashboard to be used as a Messenger. However, since the UI for messaging has been created, I have excluded the above route.